x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -14,27 +14,21 @@ import { CasesStatusResponseRt, CaseStatusRt } from './status';
 import { CaseConnectorRt } from '../connectors';
 import { CaseAssigneesRt } from './assignee';
 
-const BucketsAggs = rt.array(
-  rt.type({
-    key: rt.string,
-  })
-);
+export const AttachmentTotalsRt = rt.type({
+  alerts: rt.number,
+  userComments: rt.number,
+});
 
-export const GetCaseIdsByAlertIdAggsRt = rt.type({
```

Comment:
This type makes more sense as an internal type in the service layer so I moved it there. It's not referenced outside of cases and doesn't need to be an `io-ts` type.